CHANGELOG.next.asciidoc

```diff
@@ -104,6 +104,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Fix EC2 host.cpu.usage {pull}35717[35717]
 - Add option in SQL module to execute queries for all dbs. {pull}35688[35688]
 - Add remaining dimensions for azure storage account to make them available for tsdb enablement. {pull}36331[36331]
+- Fix Azure Resource Metrics missing metrics after upgrade to 8.11.3 {issue}37642[37642] {pull}37643[37643]
```

Comment:
```suggestion
- Fix Azure Resource Metrics missing metrics (min and max aggregations) after upgrade to 8.11.3 {issue}37642[37642] {pull}37643[37643]
```

Reply:
just in order the issue to be searcahble

Reply:
Sounds great, adding it.